### DISCUSSION TITLE
osu lazer mania custom timing based note color

### DISCUSSION THREAD

**Discussion by inukedachild** [ppy/osu#30081] [Ideas]
i am so sorry if someone asked this before i genuinley dont know what its called

anyway you know in rulesets > mania you can enable a thingy that makes the note colors change based on the beat (1st beat, 2nd beat, 4 beat ect)? yeah well make it have a lil drop down and make it customizable

**Comment by MegaGamer22** [comment]
I'll bump this because I need this too; I need the ITG colors to read them properly and the current one confuses me a lot. Probably some presets like "ITG" or "Stepmania" would be very practical.

**Comment by Ghtf47** [comment]
Bump, A lot of skins get black colored notes when too many notes are  on screen; making  them near invisible

**Reply by MegaGamer22** [reply]
You have to use a skin with white notes.
